docs/ja/plugins/plugin-toml.md

```
@@ -0,0 +1,234 @@
+
+### `[plugin.i18n]` — 多言語対応
+
+```toml
+[plugin.i18n]
+default_locale = "zh-CN"
+locales_dir = "i18n"
+```
+
+多言語対応が必要な場合、プラグインディレクトリに `i18n/` フォルダーを作り、ロケールファイルを置きます。
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)